javascript/ecmascript-2015/function-improvements/arrow-functions-part-1.md

```diff
@@ -40,44 +40,48 @@ links:
 
 **Arrow functions** (called lambda or "fat arrow" functions in some other languages) provide a shorthand syntax to declare function expression.
 
-```
-(parameters) => {}
+```js
+(parameters) => {};
 ```
 
 The parenthesis are optional if there is a single parameter:
 
-```
-parameter => {}
+```js
+parameter => {};
 ```
 
 The braces are optional if the returned expression is one line:
 
-```
-parameter => expression
+```js
+parameter => expression;
 ```
 
 Below we use arrow function syntax to declare a function that accepts two parameters (x and y) and will add them together:
 
-```
-let add = (x,y) => { x + y }
-add(2,4); //6
+```js
+let add = (x, y) => {
+  x + y;
```

Comment:
This example is incorrect and will return `undefined`.

If an arrow function has `{}`  then we have to use the `return` statement:

We should maybe show both ways of defining an arrow function here:

```js
let add = (x, y) => {
  return x + y;
}

add(2, 4) // undefined
```

or

```js
let add = (x, y) => x + y;

add(2, 4) // undefined
```

Reply:
done